### PR TITLE
ORCH-387 Replace usage of api/properties by api/settings

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java [hunk 1]
```diff
@@ -128,16 +132,29 @@ public void start() {
...
+      newAdminWsClient().settingsService().set(SetRequest.builder().setKey(key).setValue(value).build());
+    } else {
+      server.getAdminWsClient().update(new PropertyUpdateQuery(key, value));
+    }
+  }
+
+  private WsClient newAdminWsClient() {
+    return WsClientFactories.getDefault().newClient(HttpConnector.newBuilder()
+      .url(server.getUrl())
+      .credentials("admin", "admin")
```
Comment (simonbrandhof) [inline sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java]: These constants are available in `Server`.

File: sonar-orchestrator/src/main/java/com/sonar/orchestrator/config/Licenses.java [hunk 2]
```diff
@@ -75,7 +75,8 @@ private String downloadFromGithub(String pluginKey) {
   @CheckForNull
   public String get(String pluginKey) {
     if (!cache.containsKey(pluginKey)) {
-      cache.put(pluginKey, downloadFromGithub(pluginKey));
+      String value = downloadFromGithub(pluginKey);
+      cache.put(pluginKey, value);
```
Comment (simonbrandhof) [inline sonar-orchestrator/src/main/java/com/sonar/orchestrator/config/Licenses.java]: why this change ?

File: sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java [hunk 3]
```diff
@@ -128,16 +132,29 @@ public void start() {
...
+        updateSetting(licenses.licensePropertyKey(pluginKey), license);
       }
     }
 
     buildRunner = new BuildRunner(config);
   }
 
+  private void updateSetting(String key, String value) {
+    if (getServer().version().isGreaterThanOrEquals("6.1")) {
+      newAdminWsClient().settingsService().set(SetRequest.builder().setKey(key).setValue(value).build());
```
Comment (simonbrandhof) [inline sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java]: as discussed please use the old `server.getAdminWsClient().post()` instead of a new client. Sharing a single client becomes critical and should be addressed soon.

File: sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java (hunk 1 above)
Comment (julienlancelot) [inline sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java]: It's gonna be removed

File: sonar-orchestrator/src/main/java/com/sonar/orchestrator/config/Licenses.java (hunk 2 above)
Comment (julienlancelot) [inline sonar-orchestrator/src/main/java/com/sonar/orchestrator/config/Licenses.java]: I'll revert this, it's a mistake

**Comment by julienlancelot** [comment]
@simonbrandhof Feedbacks applied (and tested)
